### ISSUE TITLE
Spotlight Colors

### ISSUE THREAD

**Issue by ghost** [kevin68/The-Spotlight-Mod#7]
The spotlight doesn't seem to remember color settings/textures when the chunk is unloaded and reloaded.

Forge: 2811
This mod: 1.2.0.97

**Comment by robin4002** [comment]
Hi,
Thank for the report, but unfortunately I am unable to duplicate the bug.
Can you provide more details about the environment ? (client or server, what others mods are installed, etc.)

EDIT: I just noticed that I have the version 1.2.0.101, can you try to update ? https://minecraft.curseforge.com/projects/the-spotlight-mod/files/2495296
(Where did you find the version 1.2.0.97 O_o ?)

**Comment by ghost** [comment]
That is what version .101 shows up as ingame -shrugs-

This is on a dedicated server, running the continuity modpack. Further information I've gleaned:
-It does stay synced between players (regardless of state)
-When the gui is opened it corrects itself for all players who are in the area

I've yet to test this in singleplayer, it's possible that the bug may not be exhibited there.
Also of note is that the spotlight is in a chunkloaded area, which may or may not be causing it (using chickenchunks chunkloader).

**Comment by robin4002** [comment]
Oh, indeed, the version number is inconsistent. The number inside the file `mcmod.info` is wrong.

I think there is an incompatibility with another mod, because I could not reproduce the bug on a forge server, even on a forge + sponge server.

Unfortunately I don't really have time to investigate more, and I would like to focus on the 1.13.2 update.

**Comment by ghost** [comment]
Fair enough. I have some free time over the next day or so and will try to find out what's going on.
